app/views/templates/search/search_results_list.rb

```diff
@@ -67,18 +67,38 @@ def navigation
           global: navigation_global,
           footer: common_footer
         }
+      end
 
+      def facets_selected
+        facets_selected_items.blank? ? nil : { items: facets_selected_items }
       end
 
       private
 
+      def facets_selected_items
```

Comment:
Assignment Branch Condition size for facets_selected_items is too high. [19.05/15]<br>Method has too many lines. [13/10]